tests/python/relay/test_op_level5.py

```diff
@@ -98,7 +98,7 @@ def test_resize3d_infer_type():
 
 
 @tvm.testing.parametrize_targets
-def test_resize3d(target, ctx):
```

Comment:
This is the correct usage of `@tvm.testing.parametrize_targets`. This test also run successfully on my local machine. Could you explain what the bug is?

Reply:
Ah, there must be something wrong with the env on my machine then. I will close this PR.